### PR TITLE
add text to readme

### PR DESCRIPTION
**Pull request by piknikki**
added spicy text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 add some text
 important code
+
+# super spicy code 
+written by spicy hackorz
```
Comment (BigBike96) [inline README.md:5]: nice

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 add some text
 important code
+
+# super spicy code
```
Comment (BigBike96) [inline README.md:4]: good

File: README.md (hunk 1 above)
Comment (piknikki) [reply README.md:5]: glad you like it!!

File: README.md (hunk 2 above)
Comment (BigBike96) [reply README.md:4]: stuff

**Review by BigBike96 — APPROVED** [review]

**Pull request merged**